### PR TITLE
add udot freeway exits data page

### PR DESCRIPTION
**Pull request by gregbunce**
keep 'em coming

### CODE REVIEW

File: src/pages/products/sgid/transportation/freeway-exits.astro [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      and place name carried by the exit sign. Route number and exit number are also carried in separate fields.
+    </p>
+  </Fragment>
+
+  <Section title="More resources" slot="more-resources" titlePosition="center">
+    <GridForMoreResources>
+      <CardWithPopularLink title="UDOT Data Portal" href="https://data-uplan.opendata.arcgis.com/"
+        >Search UDOT's Geospatial Data</CardWithPopularLink
+      >
+      <CardWithPopularLink title="UDOT Homepage" href="https://www.udot.utah.gov/connect/"
```
Comment (steveoh) [inline src/pages/products/sgid/transportation/freeway-exits.astro:58]: These are sentence cased

File: src/pages/products/sgid/transportation/freeway-exits.astro [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        >Including data on current and seasonal conditions</CardWithPopularLink
```
Comment (steveoh) [inline src/pages/products/sgid/transportation/freeway-exits.astro:59]: These should have punctuation

**Review by steveoh — COMMENTED** [review]
A couple of polish items

**Comment by gregbunce** [comment]
looks good to me.

**Review by steveoh — APPROVED** [review]

**Pull request merged**